engine-rest/engine-rest/src/test/java/org/camunda/bpm/engine/rest/ProcessInstanceRestServiceInteractionTest.java

```diff
@@ -653,7 +678,59 @@ public void testDeleteAsyncHistoricQueryBasedWithSkipSubprocesses() {
       null,
       null,
       false,
-      true);
+      true,
+      false
+    );
+  }
+
+  @Test
+  public void testDeleteAsyncHistoricQueryBasedWithSkipIoMappingsTrue() {
+    when(runtimeServiceMock.deleteProcessInstancesAsync(
+        eq((List<String>)null),
+        eq((ProcessInstanceQuery)null),
+        eq((HistoricProcessInstanceQuery)null),
+        any(),
+        anyBoolean(),
+        anyBoolean(),
+        eq(true)
+    ))
+        .thenReturn(new BatchEntity());
```

Comment:
🙃 Looks a bit weird.

```suggestion
    )).thenReturn(new BatchEntity());
```

Reply:
Applied suggestion.